rbac/management/workspace/model.py

```diff
@@ -103,6 +98,27 @@ def _ancestry_queryset(self):
                JOIN ancestors a ON w.id = a.parent_id)
             SELECT id
             FROM ancestors
+            WHERE id != %s
+        """,
+        )
+        return Workspace.objects.filter(id__in=RawSQL(sql, [self.id, self.id]))
+
+    def descendants(self):
```

Comment:
Is this used anywhere or will it be used later ?

Reply:
It's not. I had initially added it and was parsing out the id values prior to the CTE, and left it for parity with ancestors, but I think we could just remove it for now as well, and reference this if/when it's needed?